python-exercises/game_5.py

```diff
@@ -0,0 +1,13 @@
+import random
+min = 1
+max = 6
```

Comment:
Those two are rather bad practise, since it shadows builtin functions (min and max).

Reply:
I see